### ISSUE TITLE
Webtorrent Desktop won't npm start

### ISSUE THREAD

**Issue by dbrowne24** [webtorrent/webtorrent-desktop#1417]
**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)
Latest version pulled from git, I believe it is 0.20.0 

**What operating system and version?**
Ubuntu 18.04

**What did you do?**
```
git clone https://github.com/webtorrent/webtorrent-desktop.git
cd webtorrent-desktop
npm install
npm start
```

**What did you expect to happen?**
webtorrent desktop should have started.

**What actually happened?**
Got the following output in the terminal followed by nothing. I have waited 30 minutes. 

```
dean@lenovo:~/Desktop/webtorrent-desktop$ npm start

> webtorrent-desktop@0.20.0 start /home/dean/Desktop/webtorrent-desktop
> npm run build && electron .


> webtorrent-desktop@0.20.0 build /home/dean/Desktop/webtorrent-desktop
> buble src --output build

Missing config file: Creating new one
(node:2504) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): ,,
(node:2504) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:2504) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): ,,
(node:2504) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 3): ,,
(node:2504) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 4): ,,
(node:2504) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 5): ,,
(node:2504) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 6): ,,
(node:2504) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 7): ,,
(node:2504) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 8): ,,
(node:2504) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 9): ,,
(node:2504) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 10): ,,
```

**Comment by dbrowne24** [comment]
Also for some reason trying to build for linux on ubuntu 18.04 leads it to only build the mac version.

**Comment by dbrowne24** [comment]
Just ran the same commands on windows 10 - same result as above

**Comment by Borewit** [comment]
I have been able to reproduce this issue on Windows 10 by cleaning remote and local configuration files first:
```bat
rd /S /Q "%LOCALAPPDATA%\WebTorrent"
rd /S /Q "%APPDATA%\WebTorrent"
```

**Comment by Borewit** [comment]
Thanks for reporting, I will look into it.

**Comment by codealchemist** [comment]
Fixed by #1419.
